apps/server/src/server.js

```diff
@@ -3,9 +3,7 @@ import 'dotenv/config'
 import { json, urlencoded } from 'body-parser'
 import express from 'express'
 import config from './config'
-// import http from './infra/http'
 import { wget as _wget, stream } from './infra/http'
-// import { wget as _wget, stream } from './infra/http/real'
 import useConnect from './connect/connectApiExpress'
 import { error as _error, info } from './infra/logger'
 // import useVcs from './incubationVcs/vcsServiceExpress'
```

Comment:
Any reason to keep this commented import? Line 9.

Reply:
Removed